File: platforms/python/examples/geometry/mesh/set_instance_count.py
from fragmentcolor import Mesh
m = Mesh()
m.add_vertices([
    [-0.01, -0.01],
    [ 0.01, -0.01],
    [ 0.00,  0.01],
])
# Draw one million instances, fetching per-particle data from a storage buffer.
m.set_instance_count(1_000_000)

File: platforms/kotlin/examples/geometry/mesh/set_instance_count.kt
import org.fragmentcolor.*
val m = Mesh()
m.addVertices(listOf(Vertex(listOf(-0.01f, -0.01f)), Vertex(listOf(0.01f, -0.01f)), Vertex(listOf(0.00f, 0.01f))))
// Draw one million instances, fetching per-particle data from a storage buffer.
m.setInstanceCount(1_000_000u)